samples/kernel/metairq_dispatch/README.rst

```diff
@@ -63,3 +63,85 @@ preemption behavior, it does not run without modification on
 native_posix platforms.  In that emulation environment, threads will
 not be preempted except at specific instrumentation points (e.g. in
 k_busy_wait()) where they will voluntarily release the CPU.
+
+Building and Running
+********************
+
+This application can be built and executed on frdm_k64f as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/kernel/metairq_dispatch
+   :host-os: unix
+   :board: frdm_k64f
+   :goals: run
+   :compact:
+
+To build for another board, change "frdm_k64f" above to that board's name.
+
+Sample Output
+=============
+
+Sample output shown below is from qemu_x86. The numbers will
```

Comment:
It's odd to use `frdm_k64f` in the build command, but show sample output from `qemu_x86`. Please be consistent with the board

Reply:
Sorry, my bad. It is by mistake that qemu_x86 is mentioned here. The command and output are both for the frdm_k64f board. Fixed this. Thanks!!